src/frontend/utils/telemetry/Instrumentation.js

```diff
@@ -1,41 +1,15 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const tracker = require('@middleware.io/agent-apm-nextjs');
 
-const opentelemetry = require('@opentelemetry/sdk-node');
-const {getNodeAutoInstrumentations} = require('@opentelemetry/auto-instrumentations-node');
-const {OTLPTraceExporter} = require('@opentelemetry/exporter-trace-otlp-grpc');
-const {OTLPMetricExporter} = require('@opentelemetry/exporter-metrics-otlp-grpc');
-const {PeriodicExportingMetricReader} = require('@opentelemetry/sdk-metrics');
-const {alibabaCloudEcsDetector} = require('@opentelemetry/resource-detector-alibaba-cloud');
-const {awsEc2Detector, awsEksDetector} = require('@opentelemetry/resource-detector-aws');
-const {containerDetector} = require('@opentelemetry/resource-detector-container');
-const {gcpDetector} = require('@opentelemetry/resource-detector-gcp');
-const {envDetector, hostDetector, osDetector, processDetector} = require('@opentelemetry/resources');
+function register() {
+    tracker.track({
+        serviceName: "frontend-project-nextjs-apm-keval",
```

Comment:
```suggestion
        serviceName: "frontend-project-nextjs-apm",
```
Should not use personal details.

Reply:
updated.